pkg/gcp/actuator/actuator.go

```diff
@@ -432,10 +432,12 @@ func (a *Actuator) syncMint(ctx context.Context, cr *minterv1.CredentialsRequest
 			roles = append(roles, role.Name)
 		} else {
 			logger.WithField("role", gcpStatus.RoleID).Info("custom role exists, updating the permissions")
```

Comment:
This log should be moved within the if statement below s.t. we only log that we are updating the role if we detect the permissions have changed.